.github/workflows/local-dev-ci.yaml

```diff
@@ -0,0 +1,60 @@
+name: Execute local test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "docs/**"
+      - "**/*.md"
+
+jobs:
+  local-dev-test:
+    name: local-dev-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: v1.17
+
+      - uses: azure/setup-kubectl@v3
+        with:
+          version: v1.23.3
+        
+      - name: Checkout kcp repository
+        uses: actions/checkout@v3
+        with:
+          repository: kcp-dev/kcp
+          ref: release-0.6
+      - name: Build KCP plugin
```

Comment:
It is building all the kcp binaries not only the kcp plugin: kcp needs to be written with lowercase.

Reply:
fixed it. it was change to `name: Build kcp binaries`